### PR TITLE
ros2 port

### PR DESCRIPTION
**Pull request by mlautman**

### CODE REVIEW

File: src/rqt_py_console/py_console_widget.py [hunk 1]
```diff
@@ -31,7 +31,7 @@
 # POSSIBILITY OF SUCH DAMAGE.
 
 import os
-import rospkg
+from ament_index_python.resources import get_resource
```
Comment (dirk-thomas) [inline src/rqt_py_console/py_console_widget.py]: The manifest should include an `exec_depend` on `ament_index_python`.

File: CMakeLists.txt [hunk 2]
```diff
@@ -1,18 +1,23 @@
...
-  DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION}
+  DESTINATION share/${PROJECT_NAME}
 )
 
 install(PROGRAMS scripts/rqt_py_console
-  DESTINATION ${CATKIN_PACKAGE_BIN_DESTINATION}
+  DESTINATION lib/${PROJECT_NAME}
 )
+
+ament_package()
```
Comment (dirk-thomas) [inline CMakeLists.txt]: Missing newline at the end of the file.

File: src/rqt_py_console/py_console_widget.py (hunk 1 above)
Comment (mlautman) [reply src/rqt_py_console/py_console_widget.py]: done

File: src/rqt_py_console/py_console_widget.py [hunk 3]
```diff
@@ -42,8 +42,11 @@ class PyConsoleWidget(QWidget):
 
     def __init__(self, context=None):
         super(PyConsoleWidget, self).__init__()
-        rp = rospkg.RosPack()
-        ui_file = os.path.join(rp.get_path('rqt_py_console'), 'resource', 'py_console_widget.ui')
+
+        _, package_path = get_resource('packages', 'rqt_py_console')
+        ui_file = os.path.join(
+            package_path, 'share', 'rqt_py_console', 'resource', 'py_console_widget.ui')
```
Comment (dirk-thomas) [inline src/rqt_py_console/py_console_widget.py]: This line needs to be fixed to address #4. It needs a directory named after the package between `resource` and the UI file.

**Pull request merged**